pkgs/applications/kde/kruler.nix

```diff
@@ -0,0 +1,17 @@
+{
+  mkDerivation, lib,
+  extra-cmake-modules, kdoctools,
+  knotifications, kwindowsystem, kxmlgui, qtx11extras
+}:
+
+mkDerivation {
+  name = "kruler";
+  meta = {
+    license = with lib.licenses; [ gpl2 ];
+    maintainers = [ lib.maintainers.fridh ];
```

Comment:
This is not you, are you sure you meant to add them as a maintainer?

Reply:
I've changed maintainer to me.